__tests__/pet.test.js

```diff
@@ -0,0 +1,135 @@
+const Pet = require('../src/pet');
+
+describe('Pet', () => {
+    it('sets the name property', () => {
+        const pet = new Pet('Fido');
+        expect(pet.name).toEqual('Fido');
```

Comment:
Its common convention to leave a clear line between the setup part of the tests and the assertions at the end:
```
        const pet = new Pet('Fido');

        expect(pet.name).toEqual('Fido');
```

You should apply this to all the tests below